tests/timeout.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+sleep 1
+ACTION_TIMEOUT=$1
+CONTAINER_NAME=$2
+
+  for seconds in $(seq 1 ${ACTION_TIMEOUT}); do
+    if [ -z  "$(docker ps| grep ${CONTAINER_NAME})" ]; then
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2143

Also, looks like you not use `shellcheck`, `.editorconfig` and `pre-commit` how it recommended by our [style guide](https://github.com/shalb/cluster.dev/blob/master/docs/style-guide.md)

`shellcheck` usage via CLI is [simple](https://github.com/koalaman/shellcheck#from-your-terminal).
Feel free ask me if you need any help with it.

Also, our indent for shell script was [set to 4](https://github.com/shalb/cluster.dev/blob/fea06e6d3c0aa49324be8c4cad7fc890988dd7dd/.editorconfig#L11), but your editor set it to 2.

P.S. Some recommendations from `shellcheck`:
![](http://i.piccy.info/i9/bd76da3c6ff3c2839b65a46dca0d002f/1584045608/419477/1362539/Screenshot_from_2020_03_12_22_38_30.png)

Reply:
I can not use pre-commit. Problem with installing in my system. 
All proposed by shellcheck fixes are not relevant. Besides the syntax error in the word. But fixed.